configure.ac

```diff
@@ -1100,6 +1107,41 @@ else
   AC_MSG_ERROR([Invalid value ($enable_spoof_source) for enable-spoof-source])
 fi
 
+dnl ***************************************************************************
+dnl snmp destination & net-snmp headers/libraries
+dnl ***************************************************************************
+AC_MSG_CHECKING(whether to enable snmp destination driver)
+
+if test "x$with_netsnmp" = "x"; then
+   NETSNMP_CONFIG="`which net-snmp-config`"
+else
+   NETSNMP_CONFIG="$with_netsnmp/net-snmp-config"
+fi
+
+if test "x$enable_snmp_dest" = "xyes"; then
+   enable_snmp_dest="yes"
+   AC_MSG_RESULT(yes)
```

Comment:
Here we need to force that net-snmp-config is available. Otherwise, when libsnmp-dev is not installed but --enable-snmp is used, configure will pass successfully,
```
echo "  snmp destination            : ${enable_snmp_dest:=no}"
```
will report success, but there will be a compile error due to missing headers. The configure phase needs to be stopped in this case, similarly to the cmake version.

Reply:
done